docs/design/TECHNICAL_DOCUMENTATION.md

```diff
@@ -0,0 +1,142 @@
+# Design tools technical documentation
+
+Otter framework exposes a set of tools to optimize the interactions between the designers and the developers.
+The purpose of these tools is to reduce the steps between the design of the solution and the integration to the applications.
+
+## Design Token
+
+The [Design Token specification](https://design-tokens.github.io/community-group/format/) define a dialog format between the design tools (such as [Figma](https://www.figma.com)) and a style language (CSS , Sass, Less, etc...).
+The Otter Framework provides **Builder**, **Schematic** and **Command Line Interface** to generate CSS Variable and/or Sass Variable, based on the inputted Design Token files.
```

Comment:
```suggestion
The Otter Framework provides **Builders**, **Schematics** and **Command Line Interfaces** to generate CSS Variables and/or Sass Variables, based on the input Design Token files.
```

Reply:
```suggestion
The Otter Framework can help you translate a Design Token file into CSS and/or Sass variables thanks to its **Builders**, **Schematics** and **Command Line Interfaces**.
```

A link to an example or some command lines to illustrate the common usage of the feature would be nice to help a user understand how to use it. (No need to go into details - this would just be an introduction to help visualize the feature)

Reply:
thanks to the provided